testing/distributions/dirichlet_test.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+
+import numpy as np
+import mxnet as mx
+from scipy.stats import dirichlet as scipy_dirichlet
+
+from mxfusion.components.variables.runtime_variable import add_sample_dimension, array_has_samples, get_num_samples
+from mxfusion.components.distributions import Dirichlet
+from mxfusion.util.testutils import numpy_array_reshape
+from mxfusion.util.testutils import MockMXNetRandomGenerator
+
+
+@pytest.mark.usefixtures("set_seed")
+class TestDirichletDistribution(object):
+    # dtype must be float64 to ensure that sum of x_i is always 1
```

Comment:
If this is really the case it should be called out in the Dirichlet class, and preferably throw an error if float32 is attempted. But it may be okay in practice to use float32 even if they don't perfectly sum to 1. @zhenwendai ?

Reply:
Ok sounds fine if it's just a testing issue with scipy. I'm not sure how to resolve that suggested change, I think only you can do that?